src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "name": "Google BigQuery API",
+    "documentation": "https://cloud.google.com/bigquery/docs/reference/rest/v2/",
```

Comment:
A REST api could be explicitly called out rather than be _documentation_, and ditto for RPC.  For example, datastore has both [REST](https://cloud.google.com/datastore/reference/rest/) and [RPC](https://cloud.google.com/datastore/docs/reference/rpc/) interfaces.  Though I dunno how we'd surfaced them in a UI.